index.html

```
@@ -249,6 +260,66 @@ <h3>OFS REQUEST SYNTAX</h3>
+									<th>Character</th>
+									<th>Symbol</th>
+									<th>OFS Replacement</th>
+									<th>Description</th>
+								</tr>
+							</thead>
+							<tbody>
+								<tr>
+									<td>Slash</td>
+									<td><code>/</code></td>
+									<td><code>^</code></td>
+									<td>Used in BATCH &amp; TSA.SERVICE IDs (e.g. BNK/AC.EOD -> BNK^AC.EOD)</td>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)